src/dotnet/UI.Blazor/Components/SettingsPanel/Tile.razor

```diff
@@ -1,9 +1,11 @@
 @namespace ActualChat.UI.Blazor.Components
 
-<div class="tile">
+<div class="tile @Class" @onclick="Click">
```

Comment:
I think we've agreed to write `@Click` instead of `Click` in such cases - i.e. to highlight it's a computed value vs a regular string.

Reply:
done